### PR TITLE
[WIP][CPU] GenerateProposals jit_refine_anchors_kernel

### PR DESCRIPTION
**Pull request by lohika-denis-kotov**
### Details:
 - *item1*
 - *...*

### Tickets:
 - *ticket-id*

### CODE REVIEW

File: src/plugins/intel_cpu/src/nodes/kernels/stack_allocator.hpp [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+
+};
+
+class Singleton {
+public:
+    Singleton() = default;
+    Singleton(const Singleton&) = delete;
+    Singleton& operator=(const Singleton&) = delete;
+    Singleton(Singleton&&) = delete;
+    Singleton&& operator=(Singleton&&) = delete;
```
Comment (rbaranchuk-capgemini) [inline src/plugins/intel_cpu/src/nodes/kernels/stack_allocator.hpp:31]: move assignment operator still has T& as return type, not T&&

Comment (lohika-denis-kotov) [reply src/plugins/intel_cpu/src/nodes/kernels/stack_allocator.hpp:31]: It is not ready for review

File: src/tests/functional/plugin/cpu/shared_tests_instances/single_layer_tests/generate_proposals.cpp [hunk 2]
```diff
@@ -16,219 +17,275 @@ const std::vector<float> nms_threshold = { 0.699999988079071 };
...
+
+struct GenerateProposalsBenchmarkTest : ov::test::BenchmarkLayerTest<GenerateProposalsLayerTest> {};
+
+TEST_P(GenerateProposalsBenchmarkTest, GenerateProposals_Benchmark) {
+    SKIP_IF_CURRENT_TEST_IS_DISABLED()
+    run("GenerateProposals", std::chrono::milliseconds(2000), 1000);
+}
+
+INSTANTIATE_TEST_SUITE_P(
+    smoke_staticInputShape0,
```
Comment (iefode) [inline src/tests/functional/plugin/cpu/shared_tests_instances/single_layer_tests/generate_proposals.cpp:232]: It should not be in the pre-commit (`*smoke*` filter).

**Comment by dmitry-gorokhov** [comment]
Closing as outdated

**Pull request closed without merging**